docs/framework/20categories/20architecture/liquidity-networks.md

```diff
@@ -1,3 +1,24 @@
 ### Liquidity Networks
+Liquidity networks enable users to transfer already minted assets from one chain to another. With such protocols, there's no ‘bridging’ of assets, i.e., no assets move across chains. Instead, assets are exchanged between one user (liquidity providers, aka LPs) and another bridge user on different networks, resulting in a cross-chain swap. For example, a user receives the desired asset on the destination network via the LP. In return, the LP receives the user’s assets on the source network and fees for providing the desired asset. 
 
-Note: include HTLCs as a possible design approach
\ No newline at end of file
+Typically, LP funds within a liquidity network are collected and locked in a smart contract (liquidity pool) to facilitate quick cross-chain swaps. When a user initiates a transaction, the desired assets are released from the smart contract on the destination network. In essence, liquidity networks work similarly to peer-to-peer networks where LPs, acting as routers, own and distribute liquidity across chains. 
+
+The relationship between the LP and the users depends on the mechanism used by the liquidity network:
```

Comment:
It feels to me that this categorization blends the concerns relating to the asset exchange mechanism with those of the underlying messaging protocol layer. In addition, it might not capture a future protocol that offers a different hybrid permutation. Perhaps we can distinguish these a bit more through a few sentences along the following lines?: "There are two broad categories of liquidity networks. Those that are solely coordinated by the exchanging parties and those that partially or fully require the support of an underlying messaging protocol to settle transactions. The latter approaches bear additional risks relating to the security of the messaging protocol they rely on. Section x, discusses the security of different message protocol architectures."

Reply:
Agreed with this! we can then add some more details about the specific risks/security features of each approach.

Reply:
Right. I took a stab at this in the [updated version here](https://crosschainriskframework.github.io/framework/20categories/20architecture/architecture/#liquidity-networks).